pandas-stubs/core/series.pyi

```diff
@@ -2324,6 +2324,13 @@ class TimedeltaSeries(Series[Timedelta]):
         *args: Any,
         **kwargs: Any,
     ) -> TimedeltaSeries: ...
+    def cumprod(
+        self,
+        axis: AxisIndex | None = ...,
```

Comment:
Remove `None`

Reply:
Since the default argument is `None`, wouldn't that cause issues?

Reply:
According to https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.cumprod.html#pandas.Series.cumprod the default argument is 0